src/room.ts

```diff
@@ -130,11 +130,17 @@ export class Room extends EventEmitter implements Sayable {
       const data = await contactGetter(this.id)
       log.silly('Room', `contactGetter(${this.id}) resolved`)
       this.rawObj = data
+      if (!this.rawObj.MemberList) {
+         return this.ready()
+      }
       await this.readyAllMembers(this.rawObj.MemberList)
       this.obj    = this.parse(this.rawObj)
       if (!this.obj) {
         throw new Error('no this.obj set after contactGetter')
       }
+      if (!this.obj.memberList) {
+        throw new Error('no this.obj.memberLit set after contactGetter')
```

Comment:
Should not throw here.

memberList is empty should be treated a normal situation, because the data is **async**.

If this.rawObj.MemberList is `undefined`, we should init `obj.memberList` with `[]`.